lib/AppInfo/Application.php

```diff
@@ -228,7 +229,7 @@ public function register(IRegistrationContext $context): void {
 	}
 
 	public function boot(IBootContext $context): void {
-		$context->injectFn(function (IMountProviderCollection $mountProviderCollection, CacheListener $cacheListener, IGroupManager $groupManager): void {
+		$context->injectFn(function (IMountProviderCollection $mountProviderCollection, CacheListener $cacheListener, Group\Manager $groupManager): void {
```

Comment:
Is it possible to use IGroupManager&Emitter or something?
Or even drop the listen calls and listen to the events instead. Pretty sure we have typed events for groups.

Reply:
Ah I tried it, but our DI gets confused and it does not actually work. So private class it is then.